lib/cfg-grammar.y

```diff
@@ -69,21 +69,24 @@ extern struct _HostResolveOptions *last_host_resolve_options;
 extern struct _StatsOptions *last_stats_options;
 extern struct _LogRewrite *last_rewrite;
 
+#define MAIN_TOKENTYPE 1
```

Comment:
I might be mistaken, but this seems to be a define trick to avoid redefinitions in plugins.

Unfortunately, this does not work starting from Bison 3.6, because the MAIN_EMPTY token's existence depends on this macro:

```
lib/cfg-grammar.c:3469:12: error: ‘MAIN_EMPTY’ undeclared (first use in this function); did you mean ‘MAIN_EOF’?
 3469 |   yychar = MAIN_EMPTY; /* Cause a token to be read.  */
      |            ^~~~~~~~~~
      |            MAIN_EOF
```

I might be a bug in Bison 3.6, of course, I'll check it.